json_schema/1.0.0.schema.json

```diff
@@ -0,0 +1,558 @@
+          "$ref": "#/definitions/condition"
+        },
+        "required": {
+          "description": "Whether or not this dependency is required.",
+          "oneOf": [
+            {
+              "type": "boolean"
+            },
+            {
+              "$ref": "#/definitions/condition"
+            }
+          ]
+        }
+      }
+    },
+    "condition": {
+      "type": "object",
+      "description": "Condition match",
+      "additionalProperties": false,
+      "properties": {
+        "environment": {
+          "type": "object",
+          "description": "Minecraft Game Environment",
+          "additionalProperties": false,
+          "properties": {
+            "client": {
+              "type": "boolean",
+              "description": "Client-side"
+            },
+            "server": {
+              "type": "boolean",
+              "description": "Server-side"
+            }
+          }
+        }
+      }
+    },
+    "installation": {
+      "type": "object",
+      "description": "Installation information",
+      "additionalProperties": false,
+      "required": [
+        "method"
+      ],
+      "properties": {
+        "method": {
+          "type": "string",
+          "description": "The installation method to use.",
+          "enum": [
+            "placeInDirectory",
+            "versionJsonInstall",
+            "runInstaller",
+            "classpathLibrary",
+            "instanceInstall",
+            "jarMod",
+            "other"
+          ]
+        },
+        "directory": true,
+        "versionJson": true
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "method": {
+              "const": "placeInDirectory"
+            },
+            "directory": {
+              "type": "string",
+              "description": "The directory to place this in."
+            }
+          },
+          "required": [
+            "directory"
+          ]
+        },
+        {
+          "properties": {
+            "method": {
+              "const": "classpathLibrary"
+            },
+            "versionJson": {
+              "type": "object",
+              "description": "The library as written in version JSON format."
+            }
+          }
+        },
+        {
+          "properties": {
+            "method": {
+              "enum": [
+                "versionJsonInstall",
+                "runInstaller",
+                "instanceInstall",
+                "jarMod",
+                "other"
+              ]
+            },
+            "directory": false,
+            "versionJson": false
+          }
+        }
+      ]
+    }
+  },
+  "required": [
+    "schemaVersion",
+    "id",
+    "name"
+  ],
+  "properties": {
+    "$schema": true,
+    "schemaVersion": {
+      "type":        "string",
+      "description": "The version of the MODIP Metadata Format.",
+      "pattern":     "^[0-9]+(\\.[0-9]+){0,2}$"
```

Comment:
This would match for any version, e.g. `9.1.5`, though this spec has version `1.0.0`. I think that patches should not break the schema, so I’ld say something like `^1\\.0(\\.[0-9]+)?$` would be more fitting here.

Reply:
You're right, that is better. I'll change it to your regex.